src/nwp_consumer/internal/entities/modelmetadata.py

```diff
@@ -297,3 +297,30 @@ class Models:
     )
     """MetOffice's Unified Model, in the Global configuration, at a resolution of 10km."""
 
+    MO_UKV_2KM: ModelMetadata = ModelMetadata(
+        name="UKV",
```

Comment:
Can it be called "um-ukv" for consistency with the other model names? In order to be clear that its MetOffice's Unified Model in the UKV configuration